Python/Product/Analysis/scrape_module.py

```diff
@@ -412,7 +422,21 @@ def _init_restype_fromdocstring(self):
             return "return dict()"
         if typeName.startswith('('):
             return "return tuple()"
-        if 'Return a string' in first_line:
+        if typeName.startswith('date'):
+            return "return datetime(1, 1, 1)"
+        if typeName.startswith('time'):
```

Comment:
What if the type name is "timer"? We probably at least need to add word boundary checks here, and we may be at the point where it makes more sense to have a table of regex -> return value mappings.